tests/UnifiedSpecTests/ServerParameterHelper.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace MongoDB\Tests\UnifiedSpecTests;
+
+use MongoDB\Client;
+use MongoDB\Driver\Exception\CommandException;
+use MongoDB\Driver\ReadPreference;
+
+use function array_key_exists;
+
+final class ServerParameterHelper
+{
+    private const GET_PARAMETERS_NOT_ALLOWED = 8000;
```

Comment:
Do we need this constant ?

Reply:
Good point, this was left over from a previous version. I removed it.